tools/git-hooks/pre-commit-hook.js

```diff
@@ -345,6 +365,8 @@ if ( phpFiles.length > 0 ) {
 		shell: true,
 		stdio: 'inherit',
 	} );
+
+	runPHPCSLinter( phpFiles );
```

Comment:
```suggestion
	runPHPLinter( phpFiles );
```

Also I'm not convinced this is the best place for this. I'd move it up to around line 361, I think.